compiler/compile/OMRCompilation.cpp

```diff
@@ -87,6 +87,7 @@
 #include "infra/CfgEdge.hpp"                   // for CFGEdge
 #include "infra/Timer.hpp"                     // for TR_SingleTimer
 #include "infra/ThreadLocal.h"                 // for tlsDefine
+#include "infra/ILWalk.hpp"                    // for PreorderNodeIterator
```

Comment:
Let's try alphabetize these as we add them.

Reply:
changed to respect alphabetic order